plugins/qcheck-stm/test/custom_config.ml

```diff
@@ -0,0 +1,9 @@
+type 'a elt = Elt of 'a
```

Comment:
We have custom configurations for all tests now, maybe we could choose a name for that test that insists on what it tests (all configuration options?)